cli/commands.go

```diff
@@ -77,12 +77,14 @@ func CreateCluster(c *cli.Context) error {
 		image = fmt.Sprintf("%s/%s", defaultRegistry, image)
 	}
 
-	// create cluster network
-	networkID, err := createClusterNetwork(c.String("name"))
-	if err != nil {
-		return err
+	if !c.IsSet("hostnetwork") || c.Int("workers") > 0 {
```

Comment:
What about printing an error when `hostnetwork` is set and `workers > 0`? :thinking: